libs/screen/image.cpp

```diff
@@ -1289,6 +1289,8 @@ void fillTriangle(Image_ img, int x0, int y0, int x1, int y1, int x2, int y2, in
     lines[0].W = lines[1].W = lines[2].W = width(img);
     lines[0].H = lines[1].H= lines[2].H = height(img);
 
+// We have 3 diff sub-function to generate Ys of edges, each particular edge map to one of them. 
+// With this kind of method pointers storage the entrence of the methods, to avoid judge which one to call at every X.
```

Comment:
```suggestion
// We have 3 different sub-functions to generate Ys of edges, each particular edge maps to one of them. 
// Use function pointers to avoid judging which function to call at every X.
```

Small suggestion, let me know if it takes away from what you originally meant!